src/engine/renderer/tr_bsp.cpp

```diff
@@ -3766,6 +3811,75 @@ void R_LoadEntities( lump_t *l, std::string &externalEntities )
 				tr.worldDeluxeMapping = glConfig2.deluxeMapping;
 			}
 
+			bool sRGBtex = false;
+			bool sRGBcolor = false;
+			bool sRGBlight = false;
+
+			s = strstr( value, "-sRGB" );
```

Comment:
It would be better to split the arguments and parse them one by one in order, so that the result is correct if some arguments override previous ones. For splitting a string on spaces, sometimes we abuse `Cmd::Args`.

Reply:
I may prefer to postpone the revamp of that whole code that is not only dealing with sRGB stuff.

Reply:
I did a much larger revamp of `R_LoadEntities()`.